.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature Request
+about: New specs for new behavior.
+labels: 'feature'
```

Comment:
you will need to go in and manually create labels with these names at https://github.com/ConservationMetrics/gc-scripts-hub/labels